app/cache/cache.js

```diff
@@ -0,0 +1,19 @@
+const { redis } = require('../cache/redis');
+
+exports.searchCache = (resolve, root, args, file) => {
+  let keyValue = null;
+  if (file === 'albums') {
```

Comment:
this is part of your business logic, not specific from cache. I'd separate it

Reply:
ok!